.travis.yml

```diff
@@ -1,11 +1,16 @@
 dist: trusty
 sudo: false
 
-language: c
+language: python
 
-os:
-  - linux
-  - osx
+matrix:
+  include:
+    - python: 2.7
+    - python: 3.6
+    # The 'osx' environment uses the python2.7 by default.
+    # TODO: maybe also tests python3 on macOS 
```

Comment:
Testing the default Python on macOS is good enough. We test on macOS just to make sure the C code builds there.